tables/bg-chardefs.cti

```diff
@@ -0,0 +1,34 @@
+# Additional characters for Bulgarian
+# Miscellaneous
+math \x00B1 235-36 PLUS-MINUS SIGN PLUS-OR-MINUS SIGN
+math \x00BC 3456-1-256 VULGAR FRACTION ONE QUARTER No 0031 2044
+math \x00BD 3456-1-23 2 VULGAR FRACTION ONE HALF No 0031 2044 0032
+math \x00BE 3456-3-256 VULGAR FRACTION THREE QUARTERS No 0033
+math \x00D7 0-236	 MULTIPLICATION SIGN
+math \x00F7 0-256 DIVISION SIGN
+noback punctuation \x0092 3
+noback punctuation \x0097 6-36
+punctuation \x00A1 235 INVERTED EXCLAMATION MARK
+punctuation \x00ad 36
+punctuation \x00BF 236 INVERTED QUESTION MARK
+punctuation \x2011 36
+punctuation \x2014 6-36
+noback punctuation \x2018 6-236
+punctuation \x201c 236
+punctuation \x201d 356
+sign \x2122 45-2345 trademark sign
+sign \x00A2 4-14 CENT SIGN
+sign \x00A3 3456 POUND SIGN
+sign \x00A4 1246 CURRENCY SIGN
+sign \x00A7 4-234 SECTION SIGN
+sign \x00A9 45-14 COPYRIGHT SIGN
+sign \x00AE 45-1235 REGISTERED SIGN REGISTERED TRADE MARK SIGN
+sign \x00B0 46-356 DEGREE SIGN
+sign \x00B4 b4 ACUTE ACCENT 0020 0301 SPACING ACUTE
+sign \x00B5 46-134 MICRO SIGN 03BC 039C 039C
+# sign \x00A5 YEN SIGN
+punctuation \x00A6 456-1256 BROKEN BAR BROKEN VERTICAL BAR
+punctuation \x0093 236
+punctuation \x0094 356
+punctuation \x0096 36
```

Comment:
What are these?

Reply:
@egli Got things straightened up in my head now. I forgot that Latin-1 and Unicode are compatible. So, of course, it's allowed to pass these characters to Liblouis and the Bulgarian table.

However they are still from the C1 control range, so other codes should preferably be used as left/right quotation mark and minus/dash. But because U+0093, U+0094 and U+0096 may represent these symbols in certain texts it should be fine to have them in tables. Just add the comments like we discussed.

Reply:
I'm going to add the same comments as in the danish table